cognite/neat/rules/models/_rules/information_rules.py

```diff
@@ -165,7 +123,11 @@ class InformationProperty(DomainProperty):
     """
 
     # TODO: Can we skip rule_type and simply try to parse the rule and if it fails, raise an error?
-
+    class_: Class_ = Field(alias="Class")
+    property_: Property_ = Field(alias="Property")
+    value_type: ValueType_ = Field(alias="Value Type")
```

Comment:
No collision with python build-in or keyword. 
```suggestion
    value_type: ValueType = Field(alias="Value Type")
```

Reply:
Same here like in case of ParentClass